lib/maxmind/geoip2/record/traits.rb

```diff
@@ -90,6 +91,18 @@ def anonymous_vpn?
           get('is_anonymous_vpn')
         end
 
+        # This is true if the IP address belongs to an
+        # {https://en.wikipedia.org/wiki/Anycast anycast network}.
+        #
+        # This property is only available from the Country, City Plus, and
+        # Insights web services and the GeoIP2 Country, City, and Enterprise
+        # databases.
```

Comment:
Isn't this everything that uses this model now?

Reply:
Hmm, not GeoLite though?

Reply:
Oh, right. Thanks!